src/pages/Campaign/CampaignItem.tsx

```diff
@@ -149,14 +149,14 @@ export default function CampaignItem({
             <ProgressBar
               title={t`Your Trading Volume`}
               percent={percentVolume}
-              value={`${tradingVolume}/${tradingVolumeRequired}`}
+              value={`${tradingVolume}/${tradingVolumeRequired}$`}
```

Comment:
`$` should be the prefix, not suffix

Reply:
Prakhar want it.
<img width="335" alt="image" src="https://user-images.githubusercontent.com/33005392/190961528-c69f4a68-09c7-452c-90e8-6c43b6166ef1.png">

Reply:
its ok if passed QA test

Reply:
Oh... Ok